modules/network/src/main/java/org/apache/ignite/internal/network/message/ClassDescriptorMessage.java

```diff
@@ -29,6 +28,17 @@
 /** Message for the {@link ClassDescriptor}. */
 @Transferable(NetworkMessageTypes.CLASS_DESCRIPTOR_MESSAGE)
 public interface ClassDescriptorMessage extends NetworkMessage {
+    int IS_PRIMITIVE_MASK = 1;
```

Comment:
Maybe it would be more clear to write it like `1 << 1` and so on?

Reply:
This seems like a matter of taste, but why not :) Fixed